### PR TITLE
A28 update: fake content-type header and exclude -bin headers

### PR DESCRIPTION
**Pull request by markdroth**

### CODE REVIEW

File: A28-xds-traffic-splitting-and-routing.md [hunk 1]
```diff
@@ -254,8 +254,14 @@ tricky.
 Note that only [custom
 metadata](https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md#requests)
 is considered during header matching, unlike in Envoy, where all headers
-(including HTTP/2 pseudo-headers) are included. This is because pick happens
-above gRPC’s transport (HTTP/2) layer, so transport headers are not accessible.
+(including HTTP/2 pseudo-headers) are included. This is because the pick
+happens above gRPC’s transport (HTTP/2) layer, so transport headers are
+not accessible. However, we will support matching against the
+`Content-type` header; if a gRPC implementation does not add that header
```
Comment (murgatroid99) [inline A28-xds-traffic-splitting-and-routing.md:260]:
```suggestion
`content-type` header; if a gRPC implementation does not add that header
```

Header names are all lower-case

**Review by dfawley — APPROVED** [review]

**Review by ejona86 — COMMENTED** [review]
LGTM

**Comment by ejona86** [comment]
This was something that we discovered Java/C/Go were aligned on, but we didn't find it documented. It was clearly agreed but then we failed to document it.

**Comment by markdroth** [comment]
For the record, the behavior in Node is consistent as well.

**Pull request merged**